otel/conf.go

```diff
@@ -421,6 +421,11 @@ windowsperfcounters/mssql_{{.MSSQLID}}:
         action: update
         new_name: processes/cpu_time
         operations:
+          - action: delete_label_value
+            label: state
+            label_value: wait
```

Comment:
Can we keep all transformations for the `state` label together (i.e., move these lines down to just before line 434)?

Reply:
Sure